src/Pages/DonatePage.jsx

```diff
@@ -0,0 +1,312 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Container,
+  Grid,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import { ArrowDropDownOutlined } from "@material-ui/icons";
+import laptop from "../Assets/laptop.svg";
+import progress from "../Assets/progress.svg";
+import note from "../Assets/note_taking.svg";
+import discovery from "../Assets/science_discovery.svg";
+import code from "../Assets/qr_code.svg";
+
+const useStyles = makeStyles((theme) => ({
+  donationRoot: {
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(8),
+  },
+  donation: {
+    padding: theme.spacing(2),
+  },
+  donationImageStyle: {
+    height: 200,
+    width: 270,
+  },
+  donationAmountStyle: {
+    textAlign: "center",
+    fontFamily: ["roboto", "sans-serif"],
+    fontWeight: 500,
+    fontSize: 60,
+  },
+  donationDescStyle: {
+    textAlign: "center",
+    fontFamily: ["roboto", "sans-serif"],
+    fontSize: 20,
+  },
+  title: {
+    textAlign: "center",
+    margin: theme.spacing(2),
+  },
+  qrCode: {
+    height: 400,
+    width: 400,
+  },
+  donate: {
+    backgroundColor: "#2F3554",
+    paddingTop: theme.spacing(5),
+    paddingBottom: theme.spacing(5),
+  },
+  donateRight: {
+    paddingTop: theme.spacing(3),
+  },
+  donateHeading: {
+    color: "white",
+    margin: theme.spacing(3),
+  },
+  donateBody: {
+    color: "white",
+    fontSize: 20,
+    margin: theme.spacing(3),
+  },
+  donateBody2: {
+    color: "white",
+    fontSize: 16,
+    margin: theme.spacing(3),
+  },
+  faq: {
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(8),
+    marginRight: theme.spacing(6),
+    marginLeft: theme.spacing(6),
+  },
+  faqAccordionStyle: {
+    borderRadius: 4,
+    marginBottom: theme.spacing(2),
+  },
+  faqSummary: {
+    backgroundColor: theme.palette.primary.light,
+    color: "white",
+    borderRadius: 4,
+  },
+  faqSumText: {
+    fontFamily: ["roboto", "sans-serif"],
+    fontSize: 20,
+    [theme.breakpoints.down("sm")]: {
+      fontSize: 16,
+    }
+  },
+  expand: {
+    color: "white",
+    fontSize: 24,
+  },
+}));
+
+const donationLevels = [
+  {
+    name: "level 1",
+    amount: "$10",
+    donationImage: progress,
+    desc: "pay for a monthly fee for one of our technological tools",
+  },
+  {
+    name: "level 2",
+    amount: "$15",
+    donationImage: discovery,
+    desc: "fund a research participant’s time",
+  },
+  {
+    name: "level 3",
+    amount: "$30",
+    donationImage: laptop,
+    desc: "pay the filing fee for a low-income individual",
+  },
+  {
+    name: "level 4",
+    amount: "Any",
+    donationImage: note,
+    desc: "directly help support those with convictions start fresh",
+  },
+];
+
+const DonatePage = () => {
```

Comment:
This comment should technically be attached to the Volunteer file, but I had a question on the page title. Do you think it should say 'Volunteer or Donate'? It feels kinds of weird for the user to think this is a page where they can learn about volunteering and then at the bottom we hit them with asking for donations. This is just a personal preference though, what do you think?

Reply:
I create branches from staging then merging staging when it gets updated, it got lost because I reverted the update then approved the pr which polluted the following pages updates.

The final design will be different with sub-pages that are exclusive, I just need to get them into the same place to hook it up.

Reply:
![image](https://user-images.githubusercontent.com/15237338/150246454-ba00042c-ba14-4c2a-9bd8-e5a6401cf3c3.png)